services/frontend-service/src/ui/Releases.tsx

```diff
@@ -314,9 +316,25 @@ export const sortEnvironmentsByUpstream = (envs: Environment[], distance: EnvSor
 export const Releases: React.FC<any> = (props: { data: GetOverviewResponse }) => {
     const { data } = props;
     const classes = useStyles(data.environments);
+    const [searchParams] = useSearchParams();
+    let apps = Object.values(data.applications);
 
-    const keys = Object.keys(data.applications);
-    keys.sort();
+    if (searchParams.has('app')) {
+        apps = apps.filter((k) => k.name.includes(searchParams.get('app')!));
+    }
+    if (searchParams.has('team')) {
+        apps = apps.filter((k) => k.team.includes(searchParams.get('team')!));
+    }
+
+    apps.sort((a, b) => {
+        if (a.name < b.name) {
```

Comment:
Does it make sense to sort by name or should we switch to sort by team and then by name?

Reply:
I would prefer to sort by team first and then by name

Reply:
updated